components/QuickViewPopup.vue

```diff
@@ -91,7 +89,11 @@
               </div>
             </div>
             <!-- Cart button & stock info -->
-            <div v-if="variation" class="w-full-px-12 fixed md:w-full md:absolute bottom-0">
+
```

Comment:
@by-tim I'm not sure where this issue is coming from, but the focus ring on option values is being cut off on both sides of the container. 
![image](https://user-images.githubusercontent.com/8597335/108734804-71ccf880-74f5-11eb-9692-f16552770ce5.png)